fix import of config.py and episode_matcher.py

ChristophCaina opened the pull request:
Issue:
When executing the Script, I am getting an exception, that the modules could not be found:

```
Traceback (most recent call last):
  File "C:\Users\{USER}\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\mkv_episode_matcher\__main__.py", line 7, in <module>
    from .config import get_config, set_config
ImportError: attempted relative import with no known parent package
```

and
```
2024-07-25 12:04:33.584 | ERROR    | __main__:<module>:181 - An error has been caught in function '<module>', process 'MainProcess' (8100), thread 'MainThread' (23780):
Traceback (most recent call last):

> File "C:\Users\{USER}\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\mkv_episode_matcher\__main__.py", line 181, in <module>
    main()
    └ <function main at 0x0000010856D40E00>
  File "C:\Users\{USER}\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\mkv_episode_matcher\__main__.py", line 173, in main
    from .episode_matcher import process_show

ImportError: attempted relative import with no known parent package
```

after removing the `.` infront of the module, the script could be executed.

Jsakkos commented:
> Issue: When executing the Script, I am getting an exception, that the modules could not be found:
> 
> ```
> Traceback (most recent call last):
>   File "C:\Users\{USER}\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\mkv_episode_matcher\__main__.py", line 7, in <module>
>     from .config import get_config, set_config
> ImportError: attempted relative import with no known parent package
> ```
> 
> and
> 
> ```
> 2024-07-25 12:04:33.584 | ERROR    | __main__:<module>:181 - An error has been caught in function '<module>', process 'MainProcess' (8100), thread 'MainThread' (23780):
> Traceback (most recent call last):
> 
> > File "C:\Users\{USER}\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\mkv_episode_matcher\__main__.py", line 181, in <module>
>     main()
>     └ <function main at 0x0000010856D40E00>
>   File "C:\Users\{USER}\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\mkv_episode_matcher\__main__.py", line 173, in main
>     from .episode_matcher import process_show
> 
> ImportError: attempted relative import with no known parent package
> ```
> 
> after removing the `.` infront of the module, the script could be executed.

Did you install the package from pip or try to run directly from the repo?

ChristophCaina commented:
tbh: I don't remember anymore :-(
But I was also trying to load the package within another script I wrote - and there, I got the same error - which was also gone after the change.

In the end, I changed the import from a recursive path and added the package name directly.

```
from .config import  ...
from mkv_episode_matcher.config import ...
```

I think, that should solve it in several different cases independend on how you call the package?!
